doc/guide/guide-basics.rst

```diff
@@ -11,21 +11,21 @@ First things first
 
 .. warning:: Do not run QuTiP from the installation directory.
 
-To load the qutip modules, first call the import statement:
+To load the qutip modules, we must first call the import statement:
 
 .. code-block:: Python
 
    from qutip import *
 
-This will load all of the user available functions. Often, we also need to import the NumPy and Matplotlib libraries with:
+that will load all of the user available functions. Often, we also need to import the NumPy and Matplotlib libraries with:
```

Comment:
I don't think the original was incorrect. It started a new sentence, and shorter sentences are good. If the import statement is to be read as a separate clause, then putting that clause in the middle of `the import statement that will load ...` is grammatically odd at best.